### PR TITLE
SC-4632: Load & Stress Test Tool

### PR DESCRIPTION
**Pull request by sprymiker**
- Developer(s): @zyuzka  

- Ticket: https://spryker.atlassian.net/browse/SC-4632

- Release Group: https://release.spryker.com/release-groups/view/3171

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   LoadTesting       | minor (new)           |                      |
  

-----------------------------------------
### Initial Release

- Introduced load-testing tool that allows load testing on an application based on Gatling

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+language: php
+
+php:
+    - 7.2
```
Comment (alexanderM91) [inline .travis.yml]: Please update to 7.3

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "spryker-sdk/load-testing",
+  "type": "library",
+  "description": "LoadTesting module",
+  "license": "proprietary",
+  "require": {
+    "php": ">=7.2",
```
Comment (alexanderM91) [inline composer.json]: The same

**Review by alexanderM91 — APPROVED** [review]

**Comment by dereuromark** [comment]
Please dont use the word module with a library.
Also in composer, changelogs and README etc.

**Comment by dereuromark** [comment]
Are you sure the license in the master branch is correct for SprykerSDK?

**Review by dereuromark — CHANGES_REQUESTED** [review]
`/src/`
as per internal (quite new) rule must not contain anything but psr4 autoloadable files for any PHP repo of ours.
All others (e.g. templates) must be removed from here and instead be in resources/ or some other top level folder or alike.

**Comment by zyuzka** [comment]
@dereuromark Hello and thank you for your advice.

1. Module word was updated on a tool 
2. License we took from https://github.com/spryker-sdk/benchmark
3. Folder structure was updated

**Pull request merged**